benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/exceptions/FabanClientThrowable.java

```diff
@@ -0,0 +1,18 @@
+package cloud.benchflow.faban.client.exceptions;
+
+/**
+ * @author vincenzoferme.
+ */
+public class FabanClientThrowable extends Throwable {
```

Comment:
should this not be a subclass of [`Exception`](https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html)?